helm/hpcc/templates/_helpers.tpl

```diff
@@ -2507,8 +2507,8 @@ global.noResourceValidation flag.  This behavior can be overridden by the caller
 A template to output a merged environment. Pass in a list with global then local environments. Only the last specified value for each named environment variable will be output
 */}}
 {{- define "hpcc.mergeEnvironments" -}}
-{{- $result := dict "MALLOC_ARENA_MAX" 8 -}}{{- /* HPCC arena default, can be overriden by component config */ -}}
-{{- range . -}}
+{{- $result := dict "MALLOC_ARENA_MAX" .defaultArenas -}}{{- /* HPCC arena default, can be overridden by component config */ -}}
```

Comment:
for safety, shouldn't this default to something (1)?, i.e. if .defaultArenas isn't present in "." (or error).
[ And perhaps remove the version that explicitly pass "defaultAreans" 1 ]